tests/album_tests/album-delete.test.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require("chai");
+const request = require("supertest");
+const db = require("../../src/db/index");
+const app = require("../../src/app");
+
+describe("delete album", () => {
+  let album;
+  beforeEach(async () => {
+    const { rows } = await db.query(
+      "INSERT INTO Albums (name, year) VALUES($1, $2) RETURNING *",
```

Comment:
I think it would be better for you testing in this file and the update and read test files if you created an entry in the artist table first then created a linked entry in the album table so the artist_id isn't null, it would check how your code is working more realistically